src/VisualStudio/SetupInteractive/source.extension.vsixmanifest

```diff
@@ -4,6 +4,7 @@
     <Identity Id="A5E5EE8D-39AE-42FF-8BBF-53C5B09C07D7" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
     <DisplayName>Roslyn Interactive Language Services</DisplayName>
     <Description>Roslyn interactive language services.</Description>
+    <ShortcutPath>..\CommonExtensions\Microsoft\ManagedLanguages\VBCSharp\InteractiveLanguageServices</ShortcutPath>
```

Comment:
I think this VSIX (SetupInteractive) and the VSInteractiveComponents VSIX below should probably be merged into a single VSIX eventually.

@KevinH-MS I remember I had discussed this with you but can't seem to recall the conclusion :) Can these VSIXes be merged?